package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@docknetwork/sdk",
-  "version": "2.4.2",
+  "version": "2.5.0",
```

Comment:
i feel like this change isnt breaking anything so 2.4.3 could work?

Reply:
As per semver, patch version is for a non-breaking bugfix, non-breaking feature additions need a minor version change.

Reply:
fair enough, i guess i cheat a little and put some smaller patches into minor bumps. works for me